settings/default/map.lua

```diff
@@ -104,6 +104,10 @@ xi.settings.map =
     -- Modifier to apply to agro'd monster speed. 0 is the retail accurate default. Negative numbers will reduce it.
     MOB_SPEED_MOD = 0,
 
+    -- Modifier that limits how fast players can move. Bypassed by feast of swords. This does not prevent cheats, btw.
+    SPEED_CAP       = 80,
```

Comment:
I don't believe this needs a setting, nor that the final speed value needs clamped to 80. By definition when we adjust the existing speed settings value we're already going non-retail, we just need to include the settings value for the final results max value (and also stay within uint8 bounds). We shouldn't be trying to decide the speed limit there, only that we're not rolling into an overflow/underflow.